src/variant.cpp

```diff
@@ -737,7 +737,12 @@ void from_variant( const variant& var,  std::vector<char>& vo )
 {
    const auto& str = var.get_string();
    FC_ASSERT( str.size() <= 2*MAX_SIZE_OF_BYTE_ARRAYS ); // Doubled because hex strings needs two characters per byte
-   vo.resize( str.size() / 2 ); {
+   if( str.find("0x") != string::npos )
```

Comment:
What is the reason for supporting hex strings prepended with "0x"? Do you have a use case for it? Without a compelling use case, I would prefer not to support it in the `from_variant` of `std::vector<char>`.

Reply:
Sometimes, the data to be passed to `vector<char>` is the result of pre-processed task by another library. In many cases, its result are represented with `0x` prefix. Of course, prefix can be stripped out easily by `replace("0x", "")`, but it would be convenient if variant accept hex string beginning with `0x`.